src/components/carousel/carousel.js

```diff
@@ -164,7 +164,7 @@ class Carousel extends React.Component {
 
     if (newIndex === currentIndex) {
       return;
-    } else if (newIndex > currentIndex) {
+    } if (newIndex > currentIndex) {
```

Comment:
err - not sure about this?

Reply:
huh

Reply:
Maybe if it was
```
if (newIndex === currentIndex) { return; }
if (newIndex > currentIndex) {
  this.transitionDirection = NEXT;
} else {
  this.transitionDirection = PREVIOUS;
```

Reply:
yeah maybe it is ok - it highlights that the else is unecessary